recipe/meta.yaml

```diff
@@ -1,36 +1,38 @@
 {% set name = "aiosqlite" %}
 {% set version = "0.18.0" %}
+{% set sha256 = "faa843ef5fb08bafe9a9b3859012d3d9d6f77ce3637899de20606b7fc39aa213" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: faa843ef5fb08bafe9a9b3859012d3d9d6f77ce3637899de20606b7fc39aa213
+  sha256: {{ sha256 }}
 
 build:
-  noarch: python
   number: 0
-  script: {{ PYTHON }} -m pip install . -vv
+  skip: true # [ py<37 ]
+  script: {{ PYTHON }} -m pip install --no-deps --no-build-isolation . -vv
 
 requirements:
   host:
-    - python >=3.6
+    - python
     - pip
     - flit-core >=2,<4
-    - typing-extensions >=3.7.2
+    - typing-extensions =3.10.0
+    - wheel
   run:
-    - python >=3.6
-    - typing-extensions >=3.7.2
+    - python
+    - typing-extensions
```

Comment:
`typing_extensions >=4.0  # [py<38]`, see https://github.com/omnilib/aiosqlite/blob/427cecc6964340b294d3ec874665786a7d8a138c/pyproject.toml#LL12C6-L12C30

Reply:
Is 3.7 even worth bothering about now? I bumped the skip to py<38 and removed typing extensions entirely.